ci_framework/roles/override_op_image/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure directory exists
+  ansible.builtin.file:
+    path: "{{ cifmw_override_op_image_dir }}"
+    state: directory
+
+- name: Update CSVs with custom images
+  when:
+  - cifmw_openstack_svc_image is defined
+  - cifmw_openshift_kubeconfig is defined
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  block:
+    - name: Dump OpenStackControlPlane CR into a file
+      ansible.builtin.shell: |
```

Comment:
As a general consideration, please use the `ci_script` instead of shell/command (I saw some other ansible.builtin.command calls in other task files).

Reply:
Updated here. In the other task, it was used to run a  single command, should I change that to use `ci_script` as well?

Reply:
ci_script allows to get proper logging in a dedicate file as well as a reproducer script, for debugging purpose or just a reference. Using it over `command` or `shell` is really a matter of "would a log or reproducer provide valuable information?" - I'd usually push for `ci_script`, but there are cases where, indeed, it doesn't really make sense (commented a `shell` call just before actually :) ).